README.md

```diff
@@ -1,56 +1,39 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
 # Project Name
+OOP-School-Library
 
 > Description the project.
-
+An OOP School Library built with ruby
```

Comment:
- [ ] Please add a proper description to this project that you working on or at least what you did in it until now to make your README file more professional.